app/models/media_object.rb

```diff
@@ -371,6 +377,10 @@ def lending_status
     Checkout.active_for_media_object(id).any? ? "checked_out" : "available"
   end
 
+  def set_lending_period
+    self.lending_period ||= Admin::Collection.find(self.collection_id).default_lending_period
```

Comment:
I'm pretty sure there is a `.collection` convenience method.
```suggestion
    self.lending_period ||= collection.default_lending_period
```